service/src/ocall_bridge/bridge_api.rs

```diff
@@ -133,7 +146,8 @@ impl From<OCallBridgeError> for sgx_status_t {
 			OCallBridgeError::InitQuote(s) => s,
 			OCallBridgeError::GetUpdateInfo(s) => s,
 			OCallBridgeError::GetIasSocket(_) => sgx_status_t::SGX_ERROR_UNEXPECTED,
-			OCallBridgeError::SendBlockAndConfirmation(_) => sgx_status_t::SGX_ERROR_UNEXPECTED,
+			OCallBridgeError::ProposeSidechainBlock(_) => sgx_status_t::SGX_ERROR_UNEXPECTED,
+			OCallBridgeError::SendExtrinsicsToParentChain(_) => sgx_status_t::SGX_ERROR_UNEXPECTED,
```

Comment:
Heh, I also found one :-)

Reply:
Ah parentchain - yes I've only checked for sidechain 😛 another search and replace commit then